src/python/platforms/fuchsia/device.py

```diff
@@ -185,7 +193,7 @@ def create(self):
     # Fuzzing jobs that SSH into the QEMU VM need access to this env var.
     environment.set_value('FUCHSIA_PKEY_PATH', qemu_vars['pkey_path'])
     logs.log('Ready to run QEMU. Command: ' + qemu_vars['qemu_path'] + ' ' +
-             str(qemu_args))
+             ' '.join('"{}"'.format(arg) for arg in qemu_args))
```

Comment:
nit: use shlex.quote instead (https://docs.python.org/3/library/shlex.html#shlex.quote) of just putting double quotes around t

Reply:
Good call; thanks for bringing that library to my attention!